app/services/DynamoChannelTests.scala

```diff
@@ -227,11 +217,12 @@ class DynamoChannelTests(stage: String, client: DynamoDbClient) extends DynamoSe
       .map(allTests => allTests.flatMap(_.priority).maxOption.getOrElse(0))
 
   // Creates a new test, with bottom priority
-  def createTest[T <: ChannelTest[T] : Encoder : Decoder](test: T, channel: Channel): ZIO[ZEnv, DynamoError, Unit] =
+  def createTest[T <: ChannelTest[T] : Encoder : Decoder](test: T, channel: Channel): ZIO[ZEnv, DynamoError, T] =
```

Comment:
we now return the item that we're creating, so that it can be written to the audit table as well